charts/external-secrets/README.md

```diff
@@ -0,0 +1,68 @@
+# External Secrets Profile Installation Guide
```

Comment:
it might be good to provide the user a bit of context about the chart, might be just references 
around what is this chart for (external secrets management) and the components or difference with the upstream chart.